src/models/ilp.js

```diff
@@ -1,42 +1,17 @@
-/*****
```

Comment:
why is the license agreement removed?

Reply:
a totally non-intentional mistake. Or it wasn't there the file I started with. there were 2 ILP models, in the end we tested one of them and the handlers were using the other one. I will put it in right away.